src/package_manager/pip.ts

```diff
@@ -80,8 +87,10 @@ export async function runPython3PipInstall(
 		cwd: path.sep,
 	};
 	if (run_with_sudo) {
+		utils.exec("sudo", pip3ConfigCommandLine);
```

Comment:
You'll need to `await` this.

```suggestion
		await utils.exec("sudo", pip3ConfigCommandLine);
```